docs/source/daml/intro/2_Scenario.rst

```diff
@@ -0,0 +1,146 @@
+.. Copyright (c) 2019 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
+.. SPDX-License-Identifier: Apache-2.0
+
+Scenarios
+=========
+
+In this section you will test the ``Token`` model from :doc:`1_Token` using DAML's inbuilt ``scenario`` language. You'll learn about the basic features of scenarios:
+
+- Getting parties
```

Comment:
Getting parties seems kind of odd to me - I know that's the name of the function, but isn't it more like "setting up parties"?

Reply:
That's not how I think about it. The Sandbox hosts all parties so `getParty` feel like a function `Text -> Party`. However, it doesn't actually do any set-up for that party. If the party never appears in a transaction, `getParty` is a no-op.

On other ledgers (ie Canton), `getParty` may do much more. A Party may no longer just be a wrapped `Text` and not all parties may be hosted, so `getParty` is properly impure.